### PR TITLE
olympics-data

### PR DESCRIPTION
**Pull request by semintelligence**

### CODE REVIEW

**Comment by semintelligence** [comment]
Done.

**Comment by dgarijo** [comment]
@semintelligence please add to the README file who the maintainers for this w3id will be

**Comment by semintelligence** [comment]
Hope its done.

File: olympics-data/.htaccess [hunk 1]
```diff
@@ -0,0 +1,4 @@
+$1 = Athlete/Olympics/AFG/Athletics/AbdulBaserWasiqi
```
Comment (davidlehn) [inline olympics-data/.htaccess:1]: What is this for?

File: olympics-data/.htaccess [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.*
```
Comment (davidlehn) [inline olympics-data/.htaccess:3]: Do you mean to check for a ttl content type here?

**Comment by davidlehn** [comment]
It would help to add github id for maintainer(s).

File: olympics-data/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # olympics-data
  The Olympics Linked Data from 1986-2020.
  
- Contact:
- Pooja Harde: pmharde29@gmail.com
- Sarika Jain: jasarika@nitkkr.ac.in
+ ## Contact:<br>
+ ### Pooja Harde: pmharde29@gmail.com <br>
+ ### Sarika Jain: jasarika@nitkkr.ac.in
```
Comment (TallTed) [inline olympics-data/README.md:6]:
```suggestion
## Contacts
* Pooja Harde: pmharde29@gmail.com <br>
* Sarika Jain: jasarika@nitkkr.ac.in
```

**Comment by dgarijo** [comment]
@semintelligence please see unsolved comments on you PR

**Comment by dgarijo** [comment]
@semintelligence ping to address the remaining comments in this PR

**Comment by dgarijo** [comment]
Closing due to inactivity. Please reopen if appropriate

**Pull request closed without merging**